rasa/core/training/interactive.py

```diff
@@ -584,14 +610,37 @@ def _slot_history(tracker_dump: Dict[Text, Any]) -> List[Text]:
         for a in predictions
     ]
 
+    tracker = await retrieve_tracker(endpoint, sender_id)
+    evts = tracker.get("events", [])
+
+    session_actions_all = [a["name"] for a in _collect_actions(evts)]
+    session_actions_unique = list(set(session_actions_all))
+    old_actions = [action["value"] for action in choices]
+    new_actions = [
```

Comment:
this isn't being used anywhere as far as i can see, what are we doing with this?

Reply:
i assume it should be added to choices?

Reply:
yes, i accidentally deleted it when changing the code here. It's put back into choices again now.